lib/templates.rake

```diff
@@ -103,6 +107,23 @@ def update_snippet
   { :diff => diff, :status => status, :result => result }
 end
 
+def get_default_branch
+  # Always use develop on Foreman-nightly
+  return "-b develop" if Foreman::Version.new.tag == "develop"
+
+  url = URI.parse("https://api.github.com/repos/theforeman/community-templates/branches")
```

Comment:
Makes sense. I think it'll be fine - it's better to check out the branch on the commandline for repos containing submodules (otherwise switching branch can get messy) but we've no expectation of using submodules in a templates repo ;)

@domcleal hah, that was a while ago. I may steal some of my own code :P